src/styles.scss

```diff
@@ -1467,6 +1467,21 @@ hr {
   margin-top: 1.5rem;
 }
 
+// Surface Material primary color for use in non-Angular components/HTML
+.primary-color {
+  color: mat-color($dockstore-app-primary);
+}
+
+// To indicate success of an operation.
+.success-color {
+  color: #00bfa5;
+}
+
+// For highlighting or distinguishing part of a title or header, e.g., "Version 0.14" in "Request DOI: Version 0.14"
+.blue-highlight {
+  color: #3f51b5;
```

Comment:
A few things I just recently noticed in the mock albeit a little too late.
The icons, the button, and part of the dialog title all use the same color.
This color is actually accent-1 success (dark) on https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/5fc699d068c46f17551f745b

Now it's still unclear when we use this color vs primary or secondary, but you can rename this class to accent-1-dark (maybe also add the entire row as a mat-color).

Afterwards, the button, icons and partial title probably should be made a consistent color (at least on that dialog alone)

Reply:
> Afterwards, the button, icons and partial title probably should be made a consistent color (at least on that dialog alone)

It seems weird to have a non-primary color for the primary button. And the mat-button doesn't seem to easily allow non-theme colors to work, although I can explore more or not use a mat-button.

I have so far made changes to sync the title and icons, and I have brought in the entire row as a mat-color. But still leaving the primary button in the primary color.

![Screen Shot 2021-06-03 at 9 34 49 AM](https://user-images.githubusercontent.com/1049340/120680508-3eccc600-c44f-11eb-82ae-91ae7e3e3038.png)

Options:

1. Screenshot is good
2. Change title and icons to primary color
3. Use the non-primary color for the button.

Thoughts? I prefer 1, then 2, then 3.

Reply:
You can slap on whatever color you want to a material button

![image](https://user-images.githubusercontent.com/24548904/120681753-b3146300-c469-11eb-90b8-31bb360217cf.png)

3 > 2 > 1